PostHog/PostHogSessionManager.swift

```diff
@@ -143,7 +143,15 @@ import Foundation
     func touchSession() {
         guard isNotReactNative() else { return }
 
-        let timestamp = now().timeIntervalSince1970
+        let timeNow = now()
+        let timestamp = timeNow.timeIntervalSince1970
+        let lastActive = sessionLock.withLock { sessionActivityTimestamp }
```

Comment:
should we check if theres an active `sessionId`? we do on L156
not need to lock twice for getting `sessionActivityTimestamp`, reuse the `lastActive` below?